modules/gapi/samples/pipeline_modeling_tool.cpp

```diff
@@ -371,7 +380,10 @@ int main(int argc, char* argv[]) {
                     builder.addDummy(call_params, read<DummyParams>(node_fn));
                 } else if (node_type == "Infer") {
                     auto infer_params = read<InferParams>(node_fn);
-                    infer_params.config = config;
+                    RETHROW_WITH_MSG_IF_FAILED(
```

Comment:
Generally, macros are evil /=

Reply:
I'll remove this macro and leave just `try-catch` there

Reply:
Done